specifications/active/aRMT-healthkit-1.0.0.yml

```diff
@@ -0,0 +1,16 @@
+name: aRMT
+vendor: RADAR
+model: aRMT-App
+version: 1.0.0
+assessment_type: QUESTIONNAIRE
+doc: aRMT Healthkit data definition.
+data:
+  - type: HEALTHKIT_GENERIC_DATA
+    topic: healthkit_generic_data
+    value_schema: .active.healthkit.HealthKitData
+  - type: HEALTHKIT_GENERIC_TYPED_DATA
+    topic: healthkit_generic_typed_data
+    value_schema: .active.healthkit.HealthKitTypedData
+  - type: HEALTHKIT_AGGREGATED_EXERCISE_DATA
+    topic: healthkit_aggregated_exercise_data
+    value_schema: .active.healthkit.HealthKitAggregatedExerciseData
```

Comment:
Could you add a doc here? Also, apart from formatting, is there any semantic difference between `HealthKitData` and `HealthKitTypedData`?